database/migrations/2020_08_27_190241_create_lms_settings_table.php

```diff
@@ -24,6 +24,7 @@ public function up()
             $table->string('description')->nullable()->default(null);
             $table->unsignedBigInteger('user_id');
             $table->foreign('user_id')->references('id')->on('users');
+            $table->string('lti_client_id');
```

Comment:
You should use a new migration instead of modifying an existing one. 
The latest dump I got (from Sept. 5th) does not have lti_client_id field in the lms_settings table.

Reply:
I think it's manageable to add this here and to not create other migration file to alter. It does not have impact on others and site. Because we still have to go from refactored to  dev and production env. 
If it will have impact or break then I will add alter new migration for this field.

Reply:
I have added migration